### ISSUE TITLE
Issue savestates Unsupported NES added by hakchi2

### ISSUE THREAD

**Issue by 1lokolo1** [ClusterM/hakchi2#106]
Hi cluster. I compile the last build. And I check some problems about Unsupported NES games like Batman or the NES games like Air Fortress(It has tiles issues) if we put --retroarch to force hakchi to use RA emulator also don't give the NESCE gui savestates. In N64 games the save appears to make ok, but fails to load. I think this last is normal because the N64 isn't very optimized for NESCE and need many power that the NESCE can't handle. Also cannot work for DosBox core(I think also its normal) Great improvements! Thanks for all hard work.

Regards.

**Comment by NaikiTurincondeliphone** [comment]
Perdona que te pregunte sobre algo que no tiene mucho que ver, pero como ya he hablado contigo por YouTube, sé que eres español xDD

¿Cómo compilas el programa? He intentado con varios, pero no soy capaz, y creo que Visual Estudio me da error al instalarlo

**Comment by 1lokolo1** [comment]
Este no es el lugar para eso. De todas formas, yo para compilarlo he usado Visual Studio 2015, el gratuito.

**Comment by NaikiTurincondeliphone** [comment]
Sé que no lo es, aun así, te doy las gracias por responder.

**Comment by ClusterM** [comment]
Which emulator you used?

**Comment by 1lokolo1** [comment]
Hi, I think the issue was resolved in your last retroarch module update. I tried today some games NES unsupported like Batman Return of the Joker(added by hakchi2) for using RetroArch(i think the core you used for this is FCEUmm. ) and works like a Charm. Also I checked games that are adding before 2.12 like Air Fortress( I put for these at the end of the command line --retroarch) saves now OK.

Thanks so much!
